src/pages/Vaults.tsx

```diff
@@ -137,6 +138,30 @@ export function Vaults() {
     error,
   } = useSWR<AxiosResponse, AxiosError>(VAULTS_DASHBOARD_QUERY, subQueryFetcher);
 
+  const vaultDataResponse: VaultsDashboardResponse = vaultsData?.data.data;
+  const [vaultsDataAppended, setVaultsDataAppended] = useState<VaultsDashboardResponse | null>(null);
+
+  const totalVaultsCount = vaultDataResponse?.vaults.totalCount || 1;
+  const pageCount = Math.ceil(totalVaultsCount / 100) - 1;
+  const { data: vaultsNextPages } = useSWR<AxiosResponse, AxiosError>(
+    pageCount ? OPEN_VAULTS_NEXT_PAGES_QUERY(pageCount) : null,
+    subQueryFetcher,
+  );
+  useEffect(() => {
+    if ((pageCount !== 0 && !vaultsNextPages) || !vaultDataResponse) return;
+    const vaultPages: {
+      [key: string]: {
+        nodes: Array<VaultsNode>;
+      };
+    } = vaultsNextPages?.data.data || {};
+    const nextVaults = Object.values(vaultPages).flatMap((openVaultsPage) => openVaultsPage.nodes);
```

Comment:
I think it's better to do: `vaultsNextPages?.data?.data
![image](https://github.com/agoric-labs/agoric-inter-dashboard/assets/60459172/eddd2e37-f0cc-4adf-a641-54c1b9b065c7)